src/Domain/Behaviour/BehaviourGateway.php

```diff
@@ -230,4 +230,28 @@ public function queryBehaviourRecordsByPerson(QueryCriteria $criteria, $gibbonSc
 
             return $this->runQuery($query, $criteria);
     }
+
+    public function getBehaviourID($gibbonSchoolYearID, $gibbonBehaviourID)
```

Comment:
It looks like, since more than just an ID is being retrieved, a good name for this method might be `getBehaviourDetails` along with `getBehaviourDetailsByCreator` below.